athlinks-ontology.ttl

```diff
@@ -258,6 +258,12 @@ dc:date rdf:type owl:AnnotationProperty .
                    rdfs:comment "An abstract role that encompasses all roles around athletic events."@en ;
                    rdfs:label "Athletic event role"@en .
 
+###  https://adp.athlinks.com/schema/AthleticEventEntry
+:AthleticEventEntry rdf:type owl:Class ;
```

Comment:
I think this is OK, but I would remove the existing athlinks_Athlete, since it was intended to play this role.